README.md

```diff
@@ -0,0 +1,599 @@
+# VCR.py
```

Comment:
This new `README.md` appears to be large chunks of `docs/` pulled together.  Sifting through it, I can only see one small note regarding `pytest.httpbin`, which probably should also have an equivalent change to `docs/`.
Anyway, im a bit lost about this new file.